<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogReserva</class>
 <widget class="QDialog" name="DialogReserva">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Número de diárias:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboTipoQuartoReserva">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboDescontoReserva">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>410</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateCheckinReserva">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome do cliente:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinDiariasReserva">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CPF do cliente:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReservar">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>470</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Reservar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Localidade:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNomeReserva">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCPFReserva">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textResumoReserva">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>170</y>
     <width>271</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Data de Checkin:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo de quarto:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Desconto aplicado (%):</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboLocalidadeReserva">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirmar">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>510</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
